packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -151,22 +167,20 @@ export const logDislikeGeneratedAnswer = (): CustomAction =>
         generativeQuestionAnsweringId,
       });
     },
-    analyticsType: 'Qna.SubmitFeedback',
-    analyticsPayloadBuilder: (state): Qna.SubmitFeedback => {
+    analyticsType: 'Qna.AnswerAction',
+    analyticsPayloadBuilder: (state): Qna.AnswerAction => {
       return {
+        action: 'dislike',
         answer: {
           responseId: state.search?.response.searchUid || '',
           type: RGAType,
         },
-        feedback: {
-          liked: false,
-        },
       };
     },
   });
 
 export const logGeneratedAnswerFeedback = (
```

Comment:
@erocheleau that s what i didi first but i got feedbacks from @fbeaudoincoveo that suggestes this, and I tend to agree with him, This way in V3, instead of removing an analytics action, we will be restricting the type this action accepts as an argument.